heartbeat/azure-phoenix.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/python
```

Comment:
Just a note on Python 2 vs. 3: Python 2 will be unmaintained by upstream by 2020, which is not that far away anymore :( I know at least two major Linux distributions that are moving towards not even providing Python 2 anymore. So for any new code written today, I would recommend at least making it Python 3-compatible, if not Python 3 only.

See: 
* https://pythonclock.org/
* https://docs.python.org/3/howto/pyporting.html

Reply:
I'll add some logic to be able to change this.

Reply:
I was hoping that we could have the same build system as we do for the fence-agents. But I am by no means familiar with automake. @oalbrigt Would it be possible to use the same build system or to add python support for the resource-agent repo as well?